pkg/psmdb/mongos.go

```diff
@@ -23,6 +23,7 @@ func MongosDeployment(cr *api.PerconaServerMongoDB) *appsv1.Deployment {
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      cr.MongosNamespacedName().Name,
 			Namespace: cr.MongosNamespacedName().Namespace,
+			Labels:    cr.Spec.Sharding.Mongos.Labels,
```

Comment:
i don't think that we want to pass this labels to deployment itself.
@delgod @spron-in should we pass them to pods and deployment?